### ISSUE TITLE
Assets for Space game 

### ISSUE THREAD

**Issue by Malthraz** [Bouncyrock/TaleSpire-Beta-AssetRequests#83]
It would be great to get some space themed floors, walls, doors, pillars and a few doodads. Or at least some assets that would pass.

Here is an image of what I produced in the old flash based dungeon painter studio:

https://imgur.com/a/tLSUzQL

If you look closely, you will note that it is just chests, tables, fonts, pools, but it kind of does look space themed.

Space themed games do not seem to be as popular as D&D, but there are probably quite a few keen people out there.

:thumbsup:

**Comment by jectordelan** [comment]
I need these as well. However, it looks like you trimmed out the end code that lets people give this a thumbs up, so they can't vote for it.

**Comment by Malthraz** [comment]
You are right, I am not quite sure what I am doing. I think I may have fixed it.

**Comment by jectordelan** [comment]
That got it. Added a thumbs up.
